src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -55802,16 +56158,17 @@ class Outer
         if (x0 is null) return;
         M2(x0) = y0;
         x0.ToString();
+        M2(x0).ToString();
```

Comment:
>M2(x0).ToString(); [](start = 8, length = 18)

The line above actually changes the flow sate of ```x0``` consider placing this line above it, so that both ```M2(x0)```` calls are based on the same state.